DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: covidregionaldata
-Title: Subnational Data for the COVID-19 Outbreak
+Title: covidregionaldata: Subnational data for COVID-19 epidemiology
```

Comment:
This will get bounced from CRAN - need to keep title case and drop the package:: bit .